.python-version

```diff
@@ -0,0 +1 @@
+3.8.0
```

Comment:
I don't think we should commit this. If so, I'd propose adding this to `.gitignore` so its not accidentally checked in again.